persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -88,7 +79,7 @@
       @SubExpr(
           name = "select",
           definedAs =
-              "select star? column* writeTimeColumn? from (where* perPartitionLimit? limit? orderBy*) allowFiltering?"),
+              "select column* star? ((count|min|max|avg|sum|writeTimeColumn) as?)* from (where* perPartitionLimit? limit? orderBy*) allowFiltering?"),
```

Comment:
Do we have to move `star` after `column*`?

Reply:
good catch -  changed.